torchgeo/datamodules/chesapeake.py

```diff
@@ -151,8 +151,8 @@ def center_crop(
         def center_crop_inner(sample: Dict[str, Tensor]) -> Dict[str, Tensor]:
             _, height, width = sample["image"].shape
 
-            y1 = (height - size) // 2
-            x1 = (width - size) // 2
+            y1 = round((height - size) / 2)
+            x1 = round((width - size) / 2)
```

Comment:
Pretty sure this is right. The previous logic can't really break, but it could lead to the centered image being shifted by up to 1 pixel. Now, it can be shifted by at most 0.5 pixels.